### PR TITLE
OM System brand and OM-1 camera support

### PR DESCRIPTION
**Pull request by sarunasb**
Based on kmilos:om-1 branch. using "OM System" instead of just "OM".

New "make", as in "manucfacturer" is OM Digital Solutions (like Olympus Imaging Corp., Olympus Corporation).

New "make", as in "brand" is "OM System", used instead of just "Olympus". Name "OM System" is used on the title page of OM-1 user manual, Exif MakerNote, SD card default directory, in OM Workspace software, by vendors alongside "Olympus", etc.

I tested this branch in darktable (with noise profile and white balance presets) by working on ~2000 .ORF images from OM-1.

### CODE REVIEW

**Comment by sarunasb** [comment]
@LebedevRI Why can't this be merged?

**Comment by sarunasb** [comment]
Hello, maintainers and developers,
please let me know what can be done to get this merged.
Thank you.

**Comment by esm7** [comment]
If I saw this PR earlier, I'd order my OM-1 1-2 months ago :laughing: 
I'm getting mine next week and can help testing, in case that's part of the blocker, or give a hand at anything else needed to get this merged.

**Comment by sarunasb** [comment]
I don't know what's impeding review and merge, but here is a darktable branch based on release 4.0.0: https://github.com/darktable-org/darktable/issues/11258#issuecomment-1179588323

**Comment by esm7** [comment]
I just built Darktable from this branch and I confirm it is working very well with my first OM-1 images!

**Comment by kmilos** [comment]
Indeed, [it looks like the branding has been somehow "officialised"](https://www.dpreview.com/news/1290364470/om-digital-solutions-is-removing-the-olympus-name-from-its-entire-product-portfolio), so this PR should probably obsolete [mine](https://github.com/darktable-org/rawspeed/pull/348).

In addition, it looks like you can add OM System OM-5 support by simply copying Olympus E-M5 Mark III. @sarunasb

**Comment by sarunasb** [comment]
Having this included in time for darktable 4.2 would be highly appreciated. Thank you!

**Comment by sarunasb** [comment]
Indeed, as far as `rawspeed` decoding, the newly released OM-5 is identical to Olympus E-M5 MkIII. Verified with sample RAW and DNG files from both cameras. OM-5 added to `data/cameras.xml`.

File: data/cameras.xml [hunk 1]
```diff
@@ -7387,6 +7387,42 @@
...
+		<Sensor black="254" white="4000"/>
+		<ColorMatrices>
+			<ColorMatrix planes="3">
+				<ColorMatrixRow plane="0">9488 -3984 -714</ColorMatrixRow>
+				<ColorMatrixRow plane="1">-2887 10945 2229</ColorMatrixRow>
+				<ColorMatrixRow plane="2">-137 960 5786</ColorMatrixRow>
+			</ColorMatrix>
+		</ColorMatrices>
+	</Camera>
+	<Camera make="OM Digital Solutions" model="OM-5">
```
Comment (kmilos) [inline data/cameras.xml]: Perhaps add a comment these are copied from Olympus E-M5 Mark III?

Comment (sarunasb) [reply data/cameras.xml]: Actually, this is the output of `tools/dngmeta.sh`, run on an image from OM-5. Except for make and model it happens to be identical to that of E-M5 mkIII. By 'clone' I mean the camera itself.

Comment (kmilos) [reply data/cameras.xml]: As there is no official support listed by ADC 15.0 for this model, I guess either 1) wait until it is listed (probably next version in December, thus missing dt 4.2) for this method, or 2) mention the copy route.

In any case, unlikely to be accepted by the maintainer as there is no sample at RPU, so might as well split into a separate PR...

Comment (sarunasb) [reply data/cameras.xml]: d4311a24 reverted. Perhaps it's better to add OM-5 in a separate simpler step after general support for OM System has been merged.

Comment (sarunasb) [reply data/cameras.xml]: P.S. Does ADC version go in sync with that of Camera Raw? If yes, v.15 does support OM-5.
https://helpx.adobe.com/camera-raw/kb/camera-raw-plug-supported-cameras.html

Comment (kmilos) [reply data/cameras.xml]: > P.S. Does ADC version go in sync with that of Camera Raw? If yes, v.15 does support OM-5. https://helpx.adobe.com/camera-raw/kb/camera-raw-plug-supported-cameras.html

Should do indeed, looks like the UK site Google was sending me to is/was not up to date!? https://helpx.adobe.com/uk/camera-raw/kb/camera-raw-plug-supported-cameras.html Go figure...

Comment (DE-cr) [reply data/cameras.xml]: > In any case, unlikely to be accepted by the maintainer as there is no sample at RPU, so might as well split into a separate PR...

There's OM-5 samples at RPU! (https://github.com/darktable-org/rawspeed/pull/399#issuecomment-1317613700)

**Comment by LebedevRI** [comment]
<del>
So i was going to merge this, but i'm not going to.
I'm not sure just how exactly else we can reword the message on https://raw.pixls.us/.
NO COLOR CHARTS.
Please contribute the same 5-sample set that is currently available there for that camera,
but of something else, like daylight landscape. Then i'll merge this.
</del>


Wrong issue, that was meant for OM-5, moved to https://github.com/darktable-org/rawspeed/pull/399

**Comment by LebedevRI** [comment]
@sarunasb i am sorry. this should not have taken so long. the PR arrived at an unfortunate time.

I've merged it in 51d9192ef11ff72667e55a72ecfb3596fc8748ab, but looks like PR does not recognize
the squash-merge as a successful merge of this pr, so looks like the PR will need to be just closed.

**Comment by LebedevRI** [comment]
@sarunasb thank you for the contribution!

**Comment by sarunasb** [comment]
> @sarunasb thank you for the contribution!



> @sarunasb i am sorry. this should not have taken so long. the PR arrived at an unfortunate time.
> 
> I've merged it in [51d9192](https://github.com/darktable-org/rawspeed/commit/51d9192ef11ff72667e55a72ecfb3596fc8748ab), ...

@LebedevRI Better later than never :) Thank you for your time!

**Pull request closed without merging**